provider/resources.go

```diff
@@ -18,11 +18,13 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"log"
+	"os"
 	"path"
 	"strings"
 
-	// embed package is not used directly
-	_ "embed"
+	_ "embed"                                          // embed package is not used directly
+	_ "github.com/pulumi/pulumi-databricks/provider/a" // Disable logging before terraform-provider-databricks is loaded
```

Comment:
Is module loading alphabetical?

Reply:
Module loading without explicit dependencies is technically UB, but for our purposes it's alphabetical.